libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFLoggerTests.m

```diff
@@ -627,7 +627,7 @@ - (NSString*)trimmedLogWithString:(NSString*)string {
     NSError *error = nil;
     static NSRegularExpression *regex = nil;
     if (!regex) {
-        regex = [NSRegularExpression regularExpressionWithPattern:@"^([\\d/]+ [\\d:\\.]+)\\sSalesforceSDKCoreTestApp\\[[\\d:]+\\]\\s(.*)"
+        regex = [NSRegularExpression regularExpressionWithPattern:@"^([\\d/]+ [\\d:\\.]+)\\sSalesforceSDKCoreTestApp\\[[\\w:]+\\]\\s(.*)"
```

Comment:
The root cause of the test failures is actually this pattern as the address is not aways numbers:

"2016/06/25 05:37:22.673 SalesforceSDKCoreTestApp[54149:**3c07**] WARNING com.salesforce <SFLoggerTests.m:701 -[SFLoggerTests testExtraLoggers]>: Log warning",